doc/en/components/charts/chart-features-synchronization.md

```diff
@@ -0,0 +1,116 @@
+---
+title: $PlatformShort$ Data Chart | Data Visualization Tools | Synchronization | Infragistics
+_description: Synchronize between multiple Infragistics' $PlatformShort$ charts controls including zooming, panning and crosshair events. Learn about our $ProductName$ graph synchronization capabilies!
+_keywords: $PlatformShort$ charts, data chart, synchronization, $ProductName$, Infragistics
+mentionedTypes: ['XamDataChart']
+---
+# $PlatformShort$ Chart Synchronization
+
+The $ProductName$ data chart component allows for synchronization with respect to the coordination of zooming, panning, and crosshair events between multiple charts. This can help you to visualize the same areas of multiple charts, assuming your data sources are similar or the same with respect to the axes.
+
+<code-view style="height: 500px" 
+           data-demos-base-url="{environment:dvDemosBaseUrl}" 
+           iframe-src="{environment:dvDemosBaseUrl}/charts/data-chart-chart-synchronization" 
+           alt="$PlatformShort$ Chart Synchronization Example" 
+           github-src="charts/data-chart/chart-synchronization">
+</code-view>
+
+<div class="divider--half"></div>
+
+There are four options of chart synchronization, in that you can synchronize horizontally only, vertically only, both, or you can choose not to synchronize at all, which is the default.
+
+If you want to synchronize a set of charts, you can assign them the same name to the `SyncChannel` property and then specify whether or not to synchronize the charts horizontally and/or vertically by setting the `SynchronizeHorizontally` and `SynchronizeVertically` properties to the corresponding boolean value.
+
+Note that in order to synchronize either vertically and/or horizontally, you will need to set the `IsHorizontalZoomEnabled` and/or `IsVerticalZoomEnabled` property to `true`, respectively. A synchronized chart that is dependent on another chart will still zoom regardless of this property setting.
+
```

Comment:
add a note that says to import Data Chart Interactivity module as it is demonstrated in [Chart Navigation](chart-features-navigation.md) topic

Reply:
pending